Vizzuality/marxan-cloud
# set up DB connection+ORM for API service

## Set up DB connection+ORM for API service

### Overview

TypeORM, db connection.

### Designs

N/A

### Testing instructions

The API service should still start (as part of `make start`), though as there are no entities/controllers configured yet, there is no actual data fetching happening.

### Feature relevant tickets

https://www.pivotaltracker.com/story/show/176205745

---

## Checklist before submitting


This is rebased on [unmerged changes](#12) as it depends on it.

      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
